cmd/notary/tuf.go

```diff
@@ -692,6 +692,15 @@ func getRemoteTrustServer(config *viper.Viper) string {
 	return defaultServerURL
 }
 
+func getUseNative(config *viper.Viper) bool {
+	conf := config.Get("useNative")
```

Comment:
I think this function can be reduced to `return config.GetBool("useNative")` -- we shouldn't need to parse strings for a boolean value